docs/pages/upgrading/agent-managed-updates/bootstrap-ec2-agent.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: Bootstrapping EC2 agents with "teleport-update"
+description: Describes how to set up EC2 instance by cloud-init script and install Teleport with "teleport-update enable"
+labels:
+ - conceptual
+ - platform-wide
+---
+
```

Comment:
This guide would benefit from having the structure of a Teleport how-to guide, rather than a conceptual guide, since it is a series of steps with a well-defined beginning and end.